elevenlabs/src/test-utils/gemini-mastra-conversation-strategy.ts

```diff
@@ -53,11 +50,18 @@ export class GeminiMastraConversationStrategy implements ConversationStrategy {
         });
 
         const agentPrompt = await this.readAgentPrompt();
+        
+        // Dynamically import publicAgents to avoid loading mcp-server at module initialization
+        // This allows tests to skip without triggering ESM import errors
+        // eslint-disable-next-line @nx/enforce-module-boundaries
+        const { publicAgents } = await import('mcp/mastra/mcp-server.js');
+        const agents = await publicAgents;
```

Comment:
This doesn't solve anything. I don't like the dynamic imports, I want them to work with static ones.